bluemira/geometry/parameterisations.py

```diff
@@ -1146,28 +1293,28 @@ def __init__(self, var_dict=None):
                     "x3", 6.5, lower_bound=6, upper_bound=10, descr="Outer limb radius"
                 ),
                 BoundedVariable(
-                    "z1_frac",
-                    0.5,
-                    lower_bound=0.4,
-                    upper_bound=0.8,
-                    descr="Fraction of height at which to start taper angle",
-                ),
-                BoundedVariable(
-                    "z2",
+                    "z1",
                     6.5,
                     lower_bound=6,
                     upper_bound=8,
                     descr="Height at which to stop the taper angle",
                 ),
                 BoundedVariable(
-                    "z3",
+                    "z2",
                     7,
                     lower_bound=6,
                     upper_bound=9,
                     descr="Upper/lower limb height",
                 ),
                 BoundedVariable(
-                    "r", 0.5, lower_bound=0, upper_bound=1, descr="Corner radius"
+                    "ri",
+                    0.0,
+                    lower_bound=0,
+                    upper_bound=0.2,
+                    descr="Inboard corner radius",
+                ),
+                BoundedVariable(
+                    "ro", 2, lower_bound=0, upper_bound=5, descr="Outboard corner radius"
                 ),
             ],
```

Comment:
This changes the API to this parameterisation. Please rebase with develop and check that you don't break any of the tests that might use this parameterisation with its old interface.